examples/custom.py

```diff
@@ -60,10 +65,5 @@ def visit_call(self, node: nodes.Call) -> None:
             in_class.locals[param.name] = node
 
 
-def register(linter):
```

Comment:
Not going to comment about this more, but I think there is benefit to the doc string. Certainly if they include a `param` line.

Reply:
Let's discuss it in https://github.com/PyCQA/pylint/pull/5558/files#r772717062